changelog/23077.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+database/elasticsearch: Added documentation for static roles.
```

Comment:
Not sure if we do changelogs for docs updates?

Reply:
No, we generally do not.